src/Components/LandingPage/index.css

```diff
@@ -1,12 +1,70 @@
-.visitor {
-    position: fixed;
-    bottom: 45%;
-    right: 40%;
+header {
+    width: 100vw;
+
+}
+
+.banner {
+    padding: 15px 0;
+    width: 90%;
+    min-width: 691px;
+    margin: 0 auto;
+}
+
+.main-container {
+    height: 60vh;
+    width: 90%;
+    min-height: 400px;
+    min-width: 691px;
+    padding: 5vh 0;
+    margin: 0 auto;
```

Comment:
there's a mix throughout of px, % and vh - this is probably the best example as it's all on one class. It's always worth sticking with one and keeping it as consistent as possible throughout, but I can see the use cases for each one and why you've used it

Reply:
You're absolutely right. We're going to leave it like this at the moment.